docs/geode-native-docs/serialization/data-serialization.html.md.erb

```diff
@@ -35,19 +35,18 @@ The type of serialization and its implementation can be specified entirely in th
 application, with no need to create corresponding code on the <%=product_name%> server.
 
 The native client release contains examples, written for .NET and C++, showing how a client application
-can register for serialiation of custom objects using the .NET `IPdxSerializable` or C++ `PdxSerializable` interface.
+can register for serialization of custom objects using the .NET IPdxSerializable interface or the C++ PdxSerializable interface.
 
-The examples are located in `examples/dotnet/pdxserializable` and `examples/cpp/pdxserializable`, respectively. 
+The examples are located in `examples\dotnet\pdxserializable` and `examples/cpp/pdxserializable`, respectively. 
 
 ## Setup
 
 PdxSerializable provides custom serialization to an individual class. Fields within an object can be serialized separately from the rest of the class.
 
 Setting up a class for PdxSerializable treatment requires some preparatory steps:
 
-- The class must inherit from the .NET `IPdxSerializable` of C++ `PdxSerializable` interface.
-- You must provide serialization instructions for objects of this class. Specifically, you must provide implementations of the 
-serialization and deserialization methods, which in .NET are named `ToData()` and `FromData()`, and in C++ arer named `toData()` and `fromData()`.
+- The class must inherit from the .NET IPdxSerializable interface or the C++ PdxSerializable interface.
+- You must provide serialization instructions for objects of this class. Specifically, you must implement `ToData()` and `FromData()` methods for .NET applications, and  `toData()` and `fromData()` methods in C++ applications.
```

Comment:
Should be "you must implement _the_ ToData()..."

Reply:
Should be "_or_ toData and..."

Reply:
Sure, why not...